implementations/elixir/ockam/ockam/lib/ockam/stream/client/bi_directional.ex

```diff
@@ -99,22 +75,22 @@ defmodule Ockam.Stream.Client.BiDirectional do
     PublisherRegistry.ensure_publisher(publisher_id, options)
   end
 
-  @bare_message {:struct, [return_stream: :string, message: :data, message_id: :uint]}
+  @bare_message {:struct, [return_stream: :string, message: :data]}
 
-  def encode_message(%{return_stream: stream, message: message, message_id: message_id}) do
+  def encode_message(%{return_stream: stream, message: message}) do
     {:ok, wire_message} = Ockam.Wire.encode(@transport_message_encoder, message)
 
     :bare.encode(
-      %{return_stream: stream, message: wire_message, message_id: message_id},
+      %{return_stream: stream, message: wire_message},
       @bare_message
     )
   end
 
   def decode_message(data) do
     case :bare.decode(data, @bare_message) do
-      {:ok, %{return_stream: stream, message: wire_message, message_id: message_id}, ""} ->
+      {:ok, %{return_stream: stream, message: wire_message}, ""} ->
```

Comment:
I think that it could be good to use with statement just to re-act to non-expected values.